sw/device/silicon_creator/lib/sigverify/ecdsa_p256_verify.c

```diff
@@ -0,0 +1,141 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/lib/sigverify/ecdsa_p256_verify.h"
+
+#include "sw/device/silicon_creator/lib/attestation.h"
+#include "sw/device/silicon_creator/lib/otbn_boot_services.h"
+#include "sw/device/silicon_creator/lib/sigverify/ecdsa_p256_key.h"
+
+#include "otp_ctrl_regs.h"
+
+enum {
+  kP256SignatureWords = kP256PublicKeyWords,
+  kP256SignatureCordWords = kP256SignatureWords / 2,
+};
+
+/*
+ * Shares for producing the `flash_exec` value in encoded message check. First
+ * 7 shares are generated using the `sparse-fsm-encode` script while the last
+ * share is `kSigverifySpxSuccess ^ kSigverifyFlashExec ^ kShares[0] ^ ... ^
+ * kShares[6]` so that xor'ing all shares and `kSigverifySpxSuccess` produces
+ * `kSigverifyFlashExec`, i.e. the value that unlocks flash execution.
```

Comment:
nit: I think in other comments the `kShares[0] ^ ... ^ kShares[6]` value is referred to as `kSigverifyEcdsaSuccess`, might be worth mentioning that name here for consistency.

Reply:
Updated description to clarify this is how `kSigverifyEcdsaSuccess` is defined.

Fixed.